frontend/src/pages/ChatPage.vue

```diff
@@ -96,13 +96,24 @@
       receiveMessage(event) {
         console.log('Message from server ', event.data);
         this.messages.push({
-          title: "Example",
           value: event.data
         });
+      },
+      getRoom(roomId) {
```

Comment:
[IMO]
Think this function is not `getRoom` because no return value.

Reply:
I couldn't understand it. Tell me more about it. I think this function does indeed get.  
So is there a better way to name it?

Reply:
Fixed in 35bc1ecd .